partiql-syntax/src/main/antlr/PartiQL.g4

```diff
@@ -0,0 +1,602 @@
+
```

Comment:
Nothing major, just curious: do we want to include the Table of operator precedence in the comments?

Reply:
Will do. Good idea.